packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -154,6 +154,16 @@ enum ShowValueIndicator {
 /// [RoundSliderTickMarkShape], [PaddleSliderValueIndicatorShape], and
 /// [RoundSliderOverlayShape] for examples.
 ///
+/// The track painting can be skipped by specifying 0 for [trackHeight].
+/// The thumb painting can be skipped by specifying
+/// [SliderComponentShape.noThumb] for [SliderThemeData.thumbShape].
+/// The overlay painting can be skipped by specifying
+/// [SliderComponentShape.noOverlay] for [SliderThemeData.overlayShape].
+/// The tick mark painting can be skipped by specifying
+/// [SliderTickMarkShape.noTickMark] for [SliderThemeData.tickMarkShape].
+/// The value indicator painting can be skipped by specifying the
+/// appropriate [ShowValueIndicator] for [SliderThemeData.showValueIndicator].
```

Comment:
I think maybe specify the exact value here is better `ShowValueIndicator.never`

Reply:
Any value thats not ShowValueIndicator.always could cause the thumb not to draw in certain cases. I could list them all?

Reply:
hmm I don't think listing them all is ideal. I'm actually fine leaving this as-is.